### ISSUE TITLE
Continue training using saved weights

### ISSUE THREAD

**Issue by guillem74** [keras-rl/keras-rl#192]
My question basically is:

Is there a way to save the weights/memory and use them after for more training? In my environment I want to train some model and afterwards continue this training changing some features, in order to adapt it to each situation. I think it will be accomplished maybe by saving the memory (e.g. Sequential Memory) into disk, and then load it again to call the agent with this loaded memory, and train it again.

**Comment by RaphaelMeudec** [comment]
Would be a nice feature I guess, have you been trying to explore a solution for this yet?

**Comment by guillem74** [comment]
@RaphaelMeudec I am working on that to make it nice and accessible, i'll keep you updated.

Also, for my bachelor thesis project, I've developed a way to work with multiple inputs for neural networks (image, position, velocity, etc) using keras-rl and gym for a drone environment in AirSim simulator. There were a few problems but I managed to get a solution. Right now I am testing results in comparision with the one input model (image). 

If I get meaningful results I'll try to make a PR to keras-rl to add the example and features. What does it take to be a contributor?
